functions/helpFunctions.tsx

```diff
@@ -1,8 +1,9 @@
-import { assignInWith } from 'lodash';
 import { v4 as uuidv4 } from 'uuid';
 import { ApprovalType, AssessmentType } from '../types/Types';
 import data from '../data/IT2810Høst2018.json';
 
+export const taskNums = [1, 2, 3]; // For tesing purposes
```

Comment:
The list for test purposes :) So all tasks are considered finished when these are finished. 
Aka, the button which leads to completion should be clickable when tasks 1, 2, and 3 are approved.